TMDbLib/TMDbLib.csproj

```diff
@@ -14,7 +14,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Newtonsoft.Json" Version="9.0.1" />
+    <PackageReference Include="Newtonsoft.Json" Version="11.0.2" />
```

Comment:
Is a Newtonsoft update necessary?

In my understanding, it is preferable to target as low a version as possible, in order to allow consumers more freedom in version choice.